admin_guide/encrypting_data.adoc

```diff
@@ -0,0 +1,291 @@
+[[admin-guide-encrypting-data-at-datastore]]
+= Encrypting Data at Datastore Layer
+:data-uri:
+:icons:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[encrypting-data-overview]]
+== Overview
+
+This topic shows how to enable and configure encryption of secret data at datastore layer. While
+examples use the `secrets` resource, any resource can be encrypted. Yet another candidate is
```

Comment:
Fixed; but I used `configmaps` instead of `ConfigMaps` because I'm not sure whether it works in this way.